### DISCUSSION TITLE
Starstruck Exchange ⭐

### DISCUSSION THREAD

**Discussion by fleetimee** [Schweinepriester/github-profile-achievements#470] [General]
Attach screenshot when you starred my repo, I'll star you back ASAP 

Don't forget to embed your repo link

Thank you

https://github.com/fleetimee/infinite-impermanence

**Comment by 1RB** [comment]
![image](https://user-images.githubusercontent.com/86501179/201514323-64505717-2495-4a7b-9df4-b1c77d199891.png)

Done

* https://github.com/1RB/Authworria

*Note: You can see who's actually starred your repo (e.g. "returned the favor") by appending `/stargazers` to the end of your repo's URL. For example: https://github.com/1RB/Authworria/stargazers*

**Reply by fleetimee** [reply]
Returned the favor 🎉

**Comment by kensoz** [comment]


**Reply by fleetimee** [reply]
Delivered 🎉

Congrats for your Starstruck achievement ⭐⭐

**Comment by muhammadnurulahsan** [comment]
star and ill star your repo back [link it below]
https://github.com/muhammadnurulahsan/Carpenco-Store-Client-Side
https://github.com/muhammadnurulahsan/Carpenco-Store-Server-Side

**Reply by fleetimee** [reply]
Done 🎉

**Reply by muhammadnurulahsan** [reply]
Thanks

**Reply by Rantoryu** [reply]
@muhammadnurulahsan I've starred yours.
Here's mine: https://github.com/Rantoryu/Personal_Website

**Comment by imomoe233** [comment]
https://github.com/mmwan101010/Durable-Federated-Learning-Backdoor
![image](https://user-images.githubusercontent.com/43236705/201977660-451dfbe8-97fb-4eca-b63c-4fa4b662f4b5.png)

**Reply by fleetimee** [reply]
Done

**Comment by ArielMAJ** [comment]
@fleetimee I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

<img width="937" alt="image" src="https://user-images.githubusercontent.com/69123486/221567023-ffb15f49-1d26-4851-a96c-8b27ea302bad.png">

**Comment by Rantoryu** [comment]
@fleetimee I've starred your repository.
![image](https://github.com/Schweinepriester/github-profile-achievements/assets/59120780/993b3173-3729-44e4-8626-29b275fcad55)
Can You star mine too?
https://github.com/Rantoryu/Personal_Website
